libraries/rtprocessing/helpers/filterkernel.cpp

```diff
@@ -71,6 +71,88 @@ using namespace RTPROCESSINGLIB;
 using namespace Eigen;
 using namespace UTILSLIB;
 
+//=============================================================================================================
+// DEFINE GLOBAL RTPROCESSINGLIB METHODS
+//=============================================================================================================
+
+QString RTPROCESSINGLIB::getStringForDesignMethod(const FilterKernel::DesignMethod &designMethod)
+{
+    QString designMethodString = "External";
```

Comment:
Would these if statements be better as if-elseif or a switch statement w/ default to avoid having to check the input again once the output is assigned, and having to assign a value twice in case the initial assignment is overwritten? (same for the 3 following functions)

Reply:
+1 good idea